platform/security/idp/security-idp-plugin/security-idp-plugin-namequalifier/src/test/groovy/org/codice/ddf/security/idp/plugin/namequalifier/NameQualifierPresignPluginSpec.groovy

```diff
@@ -93,12 +92,33 @@ class NameQualifierPresignPluginSpec extends Specification {
 
         then:
         1 * nameID.getFormat()
+        2 * issuer.setNameQualifier(NAME_QUALIFIER)
+
+    }
+
+    def 'test name qualifier when issuer format is null and is persistent'() {
```

Comment:
✏️ "test name qualifier when issuer is null and nameid format is persistent" ("issuer" instead of "issuer format") ("nameid format is persistent" instead of "is persistent")